py3-certifi.yaml

```diff
@@ -24,16 +24,17 @@ vars:
 data:
   - name: py-versions
     items:
-      3.10: "310"
-      3.11: "311"
-      3.12: "312"
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      # certifi uses a special versioning scheme where the version is a date like "2023.07.22". During the
+      # build process, the version is "normalized" (and importantly, changed) to a form like "2023.7.22",
+      # which creates problems with vulnerability scanners, since vulnerability data for certifi is filed
+      # using its actual version numbers (the "2023.07.22" form). This transform creates the "normalized"
+      # version, so that we can use it when needed to update the version string back to its original form in
+      # the Python package's metadata.
+      3.13: '313'
```

Comment:
Nit: probably want to put this comment outside the list

Reply:
ruamel di that too.